gui/wxpython/image2target/ii2t_manager.py

```diff
@@ -43,6 +43,7 @@
 import wx.lib.colourselect as csel
 from core import globalvar
 from wx.lib.mixins.listctrl import ColumnSorterMixin, ListCtrlAutoWidthMixin
+import pathlib as Path
```

Comment:
That's wrong. You can't create Path("somepath/somepathpart") with pathlib("somepath/somepathpart"), like this is doing. Was this gen AI? An IDE autocompletion wouldn't have suggested it. If you did it manually, then don't worry, it happens

Reply:
Fixed